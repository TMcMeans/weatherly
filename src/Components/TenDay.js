import React from "react";
import Card from "./Card";

function TenDay(props) {
  const { forecastday } = props.dailyForecast.simpleforecast;
  return (
    <div className="tenDayCards">
      {forecastday.map((day, index) => {
        return (
          <Card
            icon={day.icon}
            weekDay={day.date.weekday_short}
            highTemp={day.high.fahrenheit}
            lowTemp={day.low.fahrenheit}
            conditions={day.conditions}
            key={index}
          />
        );
      })}
    </div>
  );
}

export default TenDay;
