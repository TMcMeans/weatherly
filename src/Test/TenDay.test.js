import React from "react";
import renderer from "react-test-renderer";
import { shallow, mount } from "enzyme";
import TenDay from "../Components/TenDay";
import data from "../mockAPI.js";

describe("TenDay component", () => {
  let { forecast } = data;

  it("renders correctly", () => {
    // const tree = renderer
    //   .create(<TenDay dailyForecast={forecast} />)
    //   .toJSON();
    // expect(tree).toMatchSnapshot();
  });
});
