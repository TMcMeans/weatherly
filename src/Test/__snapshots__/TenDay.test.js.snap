// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TenDay component renders correctly 1`] = `
<div
  className="tenDayCards"
>
  <div
    className="card"
  >
    <h1>
      Wed
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Thu
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Fri
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Sat
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Sun
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Mon
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Tue
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Wed
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Thu
    </h1>
  </div>
  <div
    className="card"
  >
    <h1>
      Fri
    </h1>
  </div>
</div>
`;
